rts/Sim/Weapons/WeaponMemPool.h

```diff
@@ -9,13 +9,14 @@
 
 #include "Sim/Weapons/PlasmaRepulser.h"
 
-static constexpr size_t WMP_S = AlignUp(sizeof(CPlasmaRepulser), 4); //biggest in size
+static constexpr size_t WMP_S = AlignUp(sizeof(CPlasmaRepulser), 8); //biggest in size
+static constexpr size_t WMP_A = 8;
```

Comment:
does the new `TypesMem` helper added in this commit mean we could change the magic 8 into some sort of `TypesMem <CPlasmaRepulser, CSomeOtherWeapon, CBlabla>` by listing the types that end up in the pool (and similarly for projectile pool, etc)?

Reply:
Yep, but it's >10 types, so also pulling in all those headers in here, so I decided to not bother. Any misalignment will be catched by static asserts anyway.